packages/dashboard/src/components/eventDetailsSidebar.tsx

```diff
@@ -0,0 +1,53 @@
+// EventDetailsSidebar.tsx
+import React from "react";
+import { Drawer, Typography, Box } from "@mui/material";
+
+interface EventDetailsSidebarProps {
+  open: boolean;
+  onClose: () => void;
+  selectedEvent: {
+    messageId: string;
+    eventType: string;
+    event: string;
+    userId: string | null;
+    anonymousId: string | null;
+    processingTime: string;
+    eventTime: string;
+    traits: string;
+  } | null;
+}
+
+const EventDetailsSidebar: React.FC<EventDetailsSidebarProps> = ({
```

Comment:
This needs more styling!

Can you use a codemirror codeblock to display the traits, and monospace typogography blocks to display the remaining values?

Reply:
Traits is a json object with like email and other details. Do you want me to use codemirror to show json object?

Reply:
![3000-rutik3-dittofeed-b0l07s2gxv6 ws-us107 gitpod io_dashboard_events](https://github.com/dittofeed/dittofeed/assets/58112334/76099252-3d85-4bf3-8f9e-24a0dcba2929)